lib/devise/multi_email/models/authenticatable.rb

```diff
@@ -48,22 +48,33 @@ def email=(new_email)
       module ClassMethods
         def find_first_by_auth_conditions(tainted_conditions, opts = {})
           filtered_conditions = devise_parameter_filter.filter(tainted_conditions.dup)
-          email = filtered_conditions.delete(:email)
+          criteria = filtered_conditions.extract!(:email, :unconfirmed_email)
```

Comment:
@allenwq man, all this time I didn't realize it's the wrong timestamp column name! Thanks for being persistent... it should be `confirmed_at`. That is now fixed....